django/applications/catmaid/control/cropping.py

```diff
@@ -136,14 +179,14 @@ def get_image(self):
         except requests.exceptions.RequestException as e:
             raise ImageRetrievalError(self.path, str(e))
 
-        blob = Blob( img_data )
-        image = Image( blob )
-        # Check if the whole image should be used and cropped if necessary.
-        src_width = image.size().width()
-        src_height = image.size().height()
+        image = PILImage.open(BytesIO(img_data))
```

Comment:
It's possible that `requests.Response` objects have some built-in way of exposing their content as a file-like.

Reply:
That's true and it would be nicer. Simply removing `BytesIO` does however now work: `embedded null byte`.